### PR TITLE
New component menu

### LINKED ISSUE #11244: New Component Menu Design
Update the component menu as per design here https://docs.google.com/document/d/1hvDEBDTVKiG-mcCeFPGajyxsVVjRkLytp2PrQpo80CI/edit?usp=sharing

image

image

### PR DESCRIPTION
**Pull request by vitvakatu**
### Pull Request Description

Closes #11244

- The new dropdown contains all buttons that were present on the wheel before. (except for disabled ‘Write always’)
- The color component stays the same for now
- The visualization toggle button does not change the color depending on the state.

https://github.com/user-attachments/assets/a4d711be-5526-41aa-8e24-6be52e187844

**UPDATE**:

The ‘Edit component’ button was replaced with the ‘Help’ button, which toggles the Help tab of the right-dock panel. The behavior is displayed on the video:

https://github.com/user-attachments/assets/6da8c34a-8bc3-4be4-b0b6-982bf709e986

The ‘Help’ item in the dropdown list preserves its current behavior (opening API docs in the browser).

### Important Notes

- I failed to fix an issue with the unmatching colors of the dropdown menu and the circular menu. If somebody wants to check it out – you are welcome. For now, we just transition to pure white without any transparency to mitigate the issue.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.
- [x] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: app/gui/src/project-view/components/CircularMenu.vue [hunk 1]
```diff
@@ -22,9 +24,17 @@ const emit = defineEmits<{
   startEditingComment: []
   openFullMenu: []
   delete: []
+  createNewNode: []
 }>()
 
+const isDropdownOpened = ref(false)
 const showColorPicker = ref(false)
+const isVisualizationEnabled = ref(props.isVisualizationEnabled)
+watch(isVisualizationEnabled, (v) => emit('update:isVisualizationEnabled', v))
```
Comment (farmaazon) [inline app/gui/src/project-view/components/CircularMenu.vue]: I think this "intermediate" ref + watch is an antipattern, and even unnecessary as I see.

Comment (kazcw) [inline app/gui/src/project-view/components/CircularMenu.vue]: Why not `defineModel`?

**Review by kazcw — APPROVED** [review]

File: app/gui/src/project-view/components/CircularMenu.vue [hunk 2]
```diff
@@ -39,64 +49,75 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
...
-          @click.stop="emit('delete')"
-        />
-      </template>
-      <div v-else class="More" @pointerdown.stop="emit('openFullMenu')"></div>
-      <ToggleIcon
-        icon="eye"
-        class="slot5"
+      <SvgButton
+        name="eye"
+        class="slot1"
```
Comment (farmaazon) [inline app/gui/src/project-view/components/CircularMenu.vue]: Now I think we could name it from directions like "slot1 -> slot_S", "slot2 -> slot_SW" etc.

File: app/gui/src/project-view/components/CircularMenu.vue [hunk 3]
```diff
@@ -139,107 +187,53 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
...
   }
 }
 
-.More {
-  position: absolute;
-  left: -5.5px;
-  top: 27px;
-  width: 42px;
-  height: 17px;
+:deep(.ColorRing .gradient) {
```
Comment (farmaazon) [inline app/gui/src/project-view/components/CircularMenu.vue]: Can we just move this style to ColorRing.vue instead of :deep?

**Review by farmaazon — APPROVED** [review]

File: app/gui/src/project-view/components/CircularMenu.vue (hunk 1 above)
Comment (vitvakatu) [reply app/gui/src/project-view/components/CircularMenu.vue]: Thanks, good suggestion

Comment (vitvakatu) [reply app/gui/src/project-view/components/CircularMenu.vue]: Fixed

File: app/gui/src/project-view/components/CircularMenu.vue (hunk 3 above)
Comment (vitvakatu) [reply app/gui/src/project-view/components/CircularMenu.vue]: Done

File: app/gui/src/project-view/components/CircularMenu.vue (hunk 2 above)
Comment (vitvakatu) [reply app/gui/src/project-view/components/CircularMenu.vue]: Done

**Pull request merged**